test/routes.js

```diff
@@ -2037,3 +2037,28 @@ test('if operationName is null, it should work fine', async (t) => {
     }
   })
 })
+
+test('disable HTTP /graphql when fullWsTransport and use the WS one', async (t) => {
+  const app = Fastify()
+  const schema = `
+    type Query {
+      add(x: Int, y: Int): Int
+    }
+  `
+  app.register(GQL, {
+    schema,
+    fullWsTransport: true
+  })
+
+  const resGet = await app.inject({
+    method: 'GET',
+    url: '/graphiql'
+  })
+  t.equal(resGet.statusCode, 404)
```

Comment:
This should not be the case.

Reply:
@mcollina Yeah this is a typo, thanks! I meant the `/graphql` endpoint... the idea behind this one is that if you set the `fullWsTransport: true`  maybe you want to _force_ every operation via WS, so you don't want the HTTP endpoints to be accessible... but maybe it doesn't make sense?

Reply:
I would not do that.